draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -209,39 +209,33 @@ to do the work implied by the rest of this document.
 
 # Document Status
 
-Following the lead of HighSpeed TCP {{?RFC3649}}, alternate congestion
-control algorithms are expected to be published as "Experimental"
-RFCs until such time that the community better understands the
-solution space.
-Traditionally, the meaning of "Experimental" status
-has varied in its use and interpretation.
-
-As part of this document
-we define two classes of congestion control proposals that can be
-published with the "Experimental" status.
-The first class includes
-algorithms that are judged to be safe to deploy for best-effort
-traffic in the global Internet and further investigated in that
-environment.
-The second class includes algorithms that, while
-promising, are not yet deemed safe enough for widespread deployment as
-best-effort traffic on the Internet, but are being specified to
-facilitate investigations in simulation, testbeds, or controlled
-environments.
-The second class can also include algorithms where the
-IETF does not yet have sufficient understanding to decide whether the
-algorithm is or is not safe for deployment on the Internet.
-
-Each alternate congestion control algorithm published is required to
-include a statement in the abstract indicating whether or not the
-proposal is considered safe for use on the Internet.
-Each alternate
-congestion control algorithm published is also required to include a
-statement in the abstract describing environments where the protocol
-is not recommended for deployment.
-There can be environments where
-the protocol is deemed *safe* for use, but it is still is not *recommended*
-for use because it does not perform well for the user.
+This document applies to congestion controls that seek Experimental or
+Standards Track status. Evaluation of both cases involves the same questions,
+but with different expectations for both the answers and the degree of
+certainty it the answers.
+
+Congestion controls without experience of internet-scale deployment SHOULD seek
```

Comment:
Would prefer /congestion control algorithms/ or better /methods/ ... but let's resolve in the Abstract PR first and then apply uniformally.

Reply:
Yep, I chose congestion controller and applied that to Sec 2. Other secionts can be a different PR.